get-users/creative-guidelines.md

```diff
@@ -0,0 +1,40 @@
+# Creative Guidelines 
+
+
+## Ad Units
+
+Promotional creatives can be banners or full screen interstitials. To learn more about what types of ads we support, and how they are displayed, check out our [ad units](doc/make-money/ad-overview) documentation.
+
+## Performance
+
+You want to optimize your creatives for top performance. If you achieve a higher TTR (tap through rate) and CR (conversion rate) your creative will waste less valuable impressions.  This will result in better ads being displayed on your app, with you earning more tap credits. 
+
+## Generating Creatives
+
+Our creative generation system makes uploading promotional creatives easy! All you need to do is provide some basic information about your application. Our system will then generate creatives based on that information and screen shots of your application from the app store. Please,  no profanities allowed! 
```

Comment:
There is an extra whitespace character between the word 'Please,' and 'no'.

Also 'Please no profanities allowed' doesn't really make sense, and also makes it look like a request rather than a requirement. I would suggest 'Profanity is not allowed'.